src/app/Fake.DotNet.Cli/DotNet.fs

```diff
@@ -769,11 +769,20 @@ module DotNet =
         let cmdArgs =
             buildCommand
                 (command |> Args.fromWindowsCommandLine |> Seq.toList)
-                (args |> Args.fromWindowsCommandLine |> Seq.toList)
+                (Args.fromWindowsCommandLine args |> Seq.toList)
                 options
 
         run cmdArgs options
 
+    /// the 'exec' above with args as a single string is public and therefore can't be changed, create a second overload with takes a string list
+    let private exec2 (buildOptions: Options -> Options) (command: string) (args: string list) =
```

Comment:
same as with `buildArgs2`, can you come up with a descriptive name instead of `exec2`